amethyst_rendy/src/camera.rs

```diff
@@ -468,10 +468,10 @@ impl<'a> PrefabData<'a> for CameraPrefab {
 }
 
 /// Active camera prefab
-pub struct ActiveCameraPrefab(usize);
+pub struct ActiveCameraPrefab(Option<usize>);
```

Comment:
Is this even used anywhere? It doesn't even implement `Deserialize`, so I don't quite see how this is useful.